frontend/package.json

```diff
@@ -10,7 +10,9 @@
     "less": "^3.10.3",
     "less-loader": "^5.0.0",
     "react": "^16.10.2",
+    "react-animated-css": "^1.2.1",
     "react-app-rewired": "^2.1.3",
+    "react-bootstrap": "^1.0.0-beta.14",
```

Comment:
We don't need to use react-bootstrap, the normal bootstrap is already included so you can just use `className="row"` and `className="col"` instead of using a component for that